frame/staking/Cargo.toml

```diff
@@ -62,6 +63,7 @@ std = [
 	"sp-runtime/std",
 	"sp-staking/std",
 	"pallet-session/std",
+	"pallet-staking-reward-fn/std",
```

Comment:
We are using the pallet reward curve's `calculate_inflation` method to determine the inflation. Now required in staking pallet.